package/MDAnalysis/coordinates/DCD.py

```diff
@@ -498,7 +498,7 @@ def _read_frame(self, frame):
         ts.frame = frame
         return ts
 
-    def timeseries(self, asel, start=0, stop=-1, skip=1, format='afc'):
+    def timeseries(self, asel=None, start=0, stop=-1, skip=1, format='afc'):
```

Comment:
*skip* is deprecated and should be replaced by *step* (that works the exact same way)